src/main/java/seedu/address/logic/commands/TeachCommand.java

```diff
@@ -0,0 +1,69 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_COURSE;
+
+import java.util.List;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.course.Course;
+import seedu.address.model.course.CourseData;
+import seedu.address.model.person.predicates.TeachingCoursePredicate;
+
+/**
+ * Adds the default course for users in the address book.
+ */
+public class TeachCommand extends Command {
+    public static final String COMMAND_WORD = "teach";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sets up default course users are teaching.\n"
+            + "Parameters: "
+            + PREFIX_COURSE + "COURSE \n"
```

Comment:
let's standardise this as COURSE_CODE

Reply:
Bug number 1 is related to @raynertjx. 

In our current feature, once the default course is set up, it is not meant to be reset from the product. We can do it manually by deleting the default course at the json file.

Reply:
> let's standardise this as COURSE_CODE

fixed